src/component/common/Table/TableCell/TableCell.tsx

```diff
@@ -0,0 +1,9 @@
+import { FC } from 'react';
+import { TableCell as MUITableCell, TableCellProps } from '@mui/material';
+import { useStyles } from './TableCell.styles';
+
+export const TableCell: FC<TableCellProps> = ({ ...props }) => {
+    const { classes: styles } = useStyles();
+
+    return <MUITableCell className={styles.tableCell} {...props} />;
```

Comment:
In cases where we create a class for very few simple styles (like here, where we have a single one) I would argue that it would be easier and clearer to add the styles with the `sx` attribute and deleting the separate `TableCell.styles.ts` file. But I guess it's a matter of preference and this may just be my Tailwind bias. 

Maybe this is something we would want to align on as a team.

Reply:
Agree, another option is to put the makeStyles file into the same file as the TableCell to avoid cluttering the render function. Though I suspect we will soon need to have a discussion on emotion and the way material-ui is taking their styling.

Reply:
`sx` has worse performance. For table cell it's important. https://mui.com/system/basics/#performance-tradeoff